### PR TITLE
[DUSTY] Cleanups + updating readme

### PR DESCRIPTION
**Pull request by Muhieddine-El-Kaissi**
- Removing unused scripts + steps
- Updating Readme
- Added couple of missing imports flagged by xcode 14.3
- Run swiftformat

### CODE REVIEW

File: .github/scripts/commit_release.sh
Comment (Abdul-Moiz) [inline .github/scripts/commit_release.sh:1]: What purpose is it used and why removed now?

File: .github/workflows/release.yml
Comment (Abdul-Moiz) [inline .github/workflows/release.yml:1]: Should we not support both SPM and cocoapods?

File: Sources/Thumbprint/Components/FloatingActionButton/TextFloatingActionButton.swift [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import ThumbprintTokens
 // import ThumbprintResources
```
Comment (Abdul-Moiz) [inline Sources/Thumbprint/Components/FloatingActionButton/TextFloatingActionButton.swift:2]: Nit: no used should we remove it?

File: .github/workflows/release.yml
Comment (Muhieddine-El-Kaissi) [reply .github/workflows/release.yml:1]: we are dropping support for pod

File: .github/scripts/commit_release.sh
Comment (Muhieddine-El-Kaissi) [reply .github/scripts/commit_release.sh:1]: we used this script to release pods which is not used anymore

**Review by Abdul-Moiz — APPROVED** [review]

**Review by dwroth — APPROVED** [review]

**Pull request merged**